prov/usnic/src/usdf_ep_dgram.c

```diff
@@ -435,7 +435,7 @@ static const struct fi_domain_attr dgram_dflt_domain_attr = {
 	.control_progress = FI_PROGRESS_AUTO,
 	.data_progress = FI_PROGRESS_MANUAL,
 	.resource_mgmt = FI_RM_DISABLED,
-	.mr_mode = OFI_MR_BASIC_MAP | FI_MR_LOCAL,
+	.mr_mode = OFI_MR_BASIC_MAP | FI_MR_LOCAL | FI_MR_BASIC,
```

Comment:
I think this is causing issues. I think we end up adding this to the info structure that we give back from `getinfo`, and then it causes problems later when we try to open up a domain. The core only accepts it if it has `FI_MR_BASIC` and nothing else when version is greater than 1.5.

Reply:
Note that this is repeated twice more below for other EP types.  You may need do adjust similar to fi_alter_domain_attr() does in prov/util/src/util_attr.c.

Reply:
@bturrubiates I was seeing exactly the opposite, however.  If we *don't* set `FI_MR_BASIC`, then we can't open a domain later (this is what was happening in Open MPI).  Sorry for the slow reply -- I'm only loosely reading my email this week...

Reply:
> @bturrubiates I was seeing exactly the opposite, however. If we don't set FI_MR_BASIC, then we can't open a domain later (this is what was happening in Open MPI). Sorry for the slow reply -- I'm only loosely reading my email this week...

I think the difference is that OMPI is requesting < 1.5 and fabtests is requesting >= 1.5.